docs/notebooks/datasets/20_newsgroup_keras/alt.atheism/49960

```diff
@@ -0,0 +1,58 @@
+Atheist Resources
+Addresses of Atheist Organizations
+USA
+FREEDOM FROM RELIGION FOUNDATION
+Darwin fish bumper stickers and assorted other atheist paraphernalia are available from the Freedom From Religion Foundation in the US.
```

Comment:
@tmylk The data used in the unit tests is present in `test/test_data` folder already. The data used in the IPython notebooks is present in `docs/notebooks/datasets` folder. We are using the same data at both places so to avoid unnecessary duplication, should I use the data in `test/test_data` (i.e. set the path accordingly in the ipynb) in the ipynb notebooks as well?